src/System.Private.Xml/src/System/Xml/Core/XmlTextReaderImplAsync.cs

```diff
@@ -367,8 +367,8 @@ private Task<bool> ReadAsync_SwitchToInteractiveXmlDecl_Helper(bool finish)
 
         private async Task<int> ReadContentAsBase64_AsyncHelper(Task<bool> task, byte[] buffer, int index, int count)
         {
-            await task.ConfigureAwait(false);
-            if (!task.Result)
+            var result = await task.ConfigureAwait(false);
+            if (!result)
```

Comment:
This change is fine, but it doesn't actually change any behavior; if the task failed, the exception would have been thrown from the await.